neuralmonkey/trainers/multitask_trainer.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, List
+
+from neuralmonkey.runners.base_runner import Executable
+from neuralmonkey.trainers.generic_trainer import GenericTrainer
+
+
+# pylint: disable=too-few-public-methods
+class MultitaskTrainer:
+    """Wrapper for scheduling multitask training.
+
+    The wrapper contains a list of trainer objects. They are being
+    called in the order defined by this list thus simulating a task
+    switching schedule.
+    """
+
+    def __init__(self,
+                 trainers: List[GenericTrainer]) -> None:
+        assert trainers is not None
+
+        self.trainers = trainers
+        self.trainer_idx = len(trainers)
+
+        self.var_list = list(set().union(*[t.var_list for t in trainers])) \
```

Comment:
Proč to potřebuje tenhleten kompletní varlist?

Reply:
trainer.var_list je pozadovany v learning_utils, tak jsem to u multitasku definoval takto